### PR TITLE
RDoc-2601 Cloud: `Cloud` -> `Portal` -> `Support tab` - update entire section

### PR DESCRIPTION
**Pull request by PFYasu**
Issue link: https://issues.hibernatingrhinos.com/issue/RDoc-2601/Cloud-Cloud-Portal-Support-tab-update-entire-section

### CODE REVIEW

**Comment by PFYasu** [comment]
Please DO NOT MERGE this PR. The main feature is not yet on the production environment.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Cloud: Support
+---
+
+{NOTE: }
+
+Support can help you assemble a production cluster that's correct for your needs,
+solve technical issues, or understand more about product-specific and general issues.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:7]:
```suggestion
Support can advise on RavenDB configuration,
help solve technical issues, or listen to product-specific feedback and other general issues.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+You can also use the support tab to write us your feedback and suggestions.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:9]:
```suggestion
You can also use the Support tab to write us your feedback and suggestions.
```

Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:9]:
```suggestion
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    * [Available support options](../cloud/cloud-support#available-support-options)
+    * [Severity and response types](../cloud/cloud-support#severity-and-response-types)
+
+{NOTE/}
+
+---
+
+{PANEL: Available support options}
+
+Cloud offers two support options - **Free** and **Paid**.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:21]:
```suggestion
RavenDB Cloud offers two support options - **Free** and **Paid**.
```

Comment (gregolsky) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:21]: Not sure if you wanna say PAID here.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Cloud offers the **Free support option** for all Accounts by default.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:23]:
```suggestion
RavenDB Cloud offers the **Free support option** for all Accounts by default.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+{NOTE/}
+
+---
+
+{PANEL: Available support options}
+
+Cloud offers two support options - **Free** and **Paid**.
+
+Cloud offers the **Free support option** for all Accounts by default.  
+Our **Free support option** covers the availability of your Cloud instance services,
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:24]:
```suggestion
Our **Free support option** covers the availability of your RavenDB Cloud instance services,
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+responding to any connectivity issues, and your participation in mailing lists and forums.
+
+---
+
+The **Paid support option** gives the support you need to get your instances up and running.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:29]:
```suggestion
The **Paid support option** provides the support you need to keep your database in good shape.  
```
Not sure about this sentence in general

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+Cloud offers the **Free support option** for all Accounts by default.  
+Our **Free support option** covers the availability of your Cloud instance services,
+responding to any connectivity issues, and your participation in mailing lists and forums.
+
+---
+
+The **Paid support option** gives the support you need to get your instances up and running.  
+Additional monitoring of your system, focus on building better *RavenDB* applications, and architectural guidance.  
+We also provide bug fixes, email and phone support, and access to the *RavenDB* core developers.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:31]:
```suggestion
Prioritized bug fixes, email and phone support, and access to the *RavenDB* core developers is in the package. 
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+Our **Free support option** covers the availability of your Cloud instance services,
+responding to any connectivity issues, and your participation in mailing lists and forums.
+
+---
+
+The **Paid support option** gives the support you need to get your instances up and running.  
+Additional monitoring of your system, focus on building better *RavenDB* applications, and architectural guidance.  
+We also provide bug fixes, email and phone support, and access to the *RavenDB* core developers. 
+
+You can choose between a **Paid support option** period of **6 months** or **12 months**.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:33]:
```suggestion
You can select one of 2 terms - **6 months** or **12 months**, which differ in price.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+{INFO: }
+The costs of the **Paid support option** are:
+**40%** of hosting cost for **6 months** contract and **30%** of hosting cost for **12 months** contract.
+{INFO/}
+
+{WARNING: }
+The **Paid support option** requires at least **one production tier** instance with minimum **$79 / month** of costs.
+{WARNING/}
+
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:43]:
```suggestion
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Minor        | Non-critical errors, general questions, requests for enhancements to the Software.                                                                                                                                                                                        |
+| Normal       | The error is not critical - no data has been lost, and the system has not failed. The error has been identified and does not prevent normal operation, or the situation may be temporarily circumvented using an available workaround.                                    |
+| Critical     | Due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available or the workaround is cumbersome to use, or the Customer data is lost or destroyed, and there is no workaround currently available. |
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:60]:
```suggestion
| Critical     | Due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available, or the Customer data is lost or destroyed, and there is no workaround currently available. |
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+{INFO: }
+**Critical severity** is only available for Customers with the **Paid support option**.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:63]:
```suggestion
**Critical severity** is available only to customers with the **Paid support option**.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+The **Paid support option** requires at least **one production tier** instance with minimum **$79 / month** of costs.
+{WARNING/}
+
+For a full support options comparision please check the [Pricing](https://cloud.ravendb.net/pricing#support-options) page.  
+
+{PANEL/}
+
+{PANEL: Severity and response types}
+
+The Cloud support form offers three types of **Severity** and two types of **Response**.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:50]:
```suggestion
The Cloud support form offers three types of **Severity** and two types of **SLA**. 
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+| Normal       | The error is not critical - no data has been lost, and the system has not failed. The error has been identified and does not prevent normal operation, or the situation may be temporarily circumvented using an available workaround.                                    |
+| Critical     | Due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available or the workaround is cumbersome to use, or the Customer data is lost or destroyed, and there is no workaround currently available. |
+
+{INFO: }
+**Critical severity** is only available for Customers with the **Paid support option**.
+{INFO/}
+
+---
+
+| **Response** | **Description**                                   |
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:68]:
```suggestion
| **SLA** | **Description**                                   |
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 14]
```diff
@@ -2,84 +2,79 @@
...
-   **A**. Choose to raise a general question  
-   **B**. Choose the issue's severity: Minor, Normal, or Critical.  
-   **C**. Write your question in your own words  
-   **D**. Send us your support call
+!["Figure 2 - Support General Message"](images\portal-support-general-question-paid-support.png "Figure 2 - Support General Message")
 
+   **1**. Choose to raise a general question.  
+   **2**. Choose the issue's severity: *Minor*, *Normal*, or *Critical*.  
+   **3**. Choose the response type: *Standard* or *Immediate*.  
+   **4**. Write your question in your own words.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:31]:
```suggestion
   **4**. Enter your question
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 15]
```diff
@@ -2,84 +2,79 @@
...
-   **B**. Choose the issue's severity: Minor, Normal, or Critical.  
-   **C**. Write your question in your own words  
-   **D**. Send us your support call
+!["Figure 2 - Support General Message"](images\portal-support-general-question-paid-support.png "Figure 2 - Support General Message")
 
+   **1**. Choose to raise a general question.  
+   **2**. Choose the issue's severity: *Minor*, *Normal*, or *Critical*.  
+   **3**. Choose the response type: *Standard* or *Immediate*.  
+   **4**. Write your question in your own words.  
+   **5**. Send us your support request.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:32]:
```suggestion
   **5**. Submit your support request.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 16]
```diff
@@ -2,84 +2,79 @@
...
+
+!["Figure 3 - Support Product-Related Message"](images\portal-support-instance-related-question-paid-support.png "Figure 3 - Support Product-Related Message")
+
+   **1**. Choose to raise a Product-related question.  
+   **2**. Choose the product you relate to.  
+   **3**. Choose the issue's severity: *Minor*, *Normal*, or *Critical*.  
+   **4**. Choose the response type: *Standard* or *Immediate*.  
+   **5**. Choose the issue category.  
+   **6**. Write your question in your own words.  
+   **7**. Send us your support request.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:48]: is it a duplicate of the above?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 17]
```diff
@@ -2,84 +2,79 @@
...
 
-* 
-   **A**. Choose to raise a Product-specific question  
-   **B**. Choose the product you relate to  
-   **C**. Choose the issue's severity: Minor, Normal, or Critical.  
-   **D**. Write your question in your own words  
-   **E**. Send us your support call
+---
+
+### Request the Paid support
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:54]:
```suggestion
### Request enabling Paid support plan
```

**Review by gregolsky — COMMENTED** [review]
see my comments please

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown (hunk 6 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:29]: It's a copy-paste from previous doc.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown (hunk 3 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:21]: I would use `Production` or `Cloud Support` instead.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown (hunk 16 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:48]: Not at all. We have a few steps more for this type of support form.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown (hunk 3 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:21]: Discussed -  let's use `Cloud Support`.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown (hunk 9 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:43]: Ok - I can remove the price requirements, but I'm not sure we should skip the note that the Cloud Support is only available for accounts with minimum one prod tier cluster.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 18]
```diff
@@ -0,0 +1,71 @@
...
+|-----------|---------------------------------------------------|
+| Standard  | Reply within a day.                               |
+| Immediate | Your query will be responded to within two hours. |
+
+{INFO: }
+**Immediate SLA** is only available for Customers with the **Cloud Support option**.  
+**Immediate SLA** is not available for **Free tier** instances.
+{INFO/}
+
+{PANEL/}
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:71]: @ppekrol can you take a look please? Can you say 'immediate SLA'??

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Minor        | Non-critical errors, general questions, requests for enhancements to the Software.                                                                                                                                                     |
+| Normal       | The error is not critical - no data has been lost, and the system has not failed. The error has been identified and does not prevent normal operation, or the situation may be temporarily circumvented using an available workaround. |
+| Critical     | Due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available, or the Customer data is lost or destroyed, and there is no workaround currently available.     |
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:53]:
```suggestion
| Critical     | Due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available or the workaround is cumbersome to use, or the Customer data is lost or destroyed, and there is no workaround currently available.   |
```

**Review by gregolsky — COMMENTED** [review]
smc

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 20]
```diff
@@ -47,32 +40,32 @@ For a full support options comparision please check the [Pricing](https://cloud.
...
+**Critical severity** is available only to customers with the **Cloud Support option**.
 {INFO/}
 
 ---
 
-| **Response** | **Description**                                   |
-|--------------|---------------------------------------------------|
-| Standard     | Reply within a day.                               |
-| Immediate    | Your query will be responded to within two hours. |
+| **SLA**   | **Description**                                   |
```
[the remark this answers is not in the document]
Comment (ppekrol) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:61]: Just Response maybe?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 21]
```diff
@@ -0,0 +1,71 @@
...
+
+---
+
+| **SLA**   | **Description**                                   |
+|-----------|---------------------------------------------------|
+| Standard  | Reply within a day.                               |
+| Immediate | Your query will be responded to within two hours. |
+
+{INFO: }
+**Immediate SLA** is only available for Customers with the **Cloud Support option**.
```
Comment (ppekrol) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:67]: Immediate Response

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown (hunk 19 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:53]: Made by mistake. Fixed.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 22]
```diff
@@ -0,0 +1,71 @@
...
+
+RavenDB Cloud offers the **Free support option** for all Accounts by default.  
+Our **Free support option** covers the availability of your RavenDB Cloud instance services,
+responding to any connectivity issues, and your participation in mailing lists and forums.
+
+---
+
+The **Cloud Support option** provides the support you need to keep your database in good shape.  
+Additional monitoring of your system, focus on building better *RavenDB* applications, and architectural guidance.  
+Prioritized bug fixes, email and phone support, and access to the *RavenDB* core developers is in the package.
```
[the remark this answers is not in the document]
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:29]: Yep, should be better to detail the benefits using list.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 23]
```diff
@@ -2,84 +2,79 @@
...
+!["Figure 1 - Support Tab"](images\portal-support-tab.png "Figure 1 - Support Tab")
 
-The root selection is between submitting a support call regarding a general topic or a product-specific one.  
+The root selection is between submitting a support request regarding a **general** topic or a **product-related** one.  
 
 ---
 
-#### Submit a General Support Call  
-
-!["Support General Message"](images\portal-support-tab-002-general.png "Support General Message")
```
[the remark this answers is not in the document]
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:30]: Changed first picture.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 24]
```diff
@@ -0,0 +1,71 @@
...
+{NOTE/}
+
+---
+
+{PANEL: Available support options}
+
+RavenDB Cloud offers two support options - **Free** and **Cloud Support**.
+
+RavenDB Cloud offers the **Free support option** for all Accounts by default.  
+Our **Free support option** covers the availability of your RavenDB Cloud instance services,
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown]: "default This "
missing a point there

**Review by reebhub — APPROVED** [review]

**Review by gregolsky — APPROVED** [review]

**Pull request merged**